spec/unit/onclient_spec.rb

```diff
@@ -16,6 +16,11 @@
       expect(SgtnClient::Translation.getString("JAVA", "helloworld", "en")).to eq 'Hello world'
     end
 
+    it "GET_INVALID_LOCALE" do
+      expect(SgtnClient::Translation.getString("JAVA", "helloworld", locale: nil)).to eq 'Hello world'
+      expect(SgtnClient::Translation.getString("JAVA", "helloworld", locale: 'garbage')).to eq 'Hello world'
```

Comment:
> Since '`locale`' has a default value 'nil', `locale` parameter can be removed from input arguments.

I do not get this? Where do you want to remove 'locale' argument from?

Reply:
Do you mean `expect(SgtnClient::Translation.getString("JAVA", "helloworld")).to eq 'Hello world'` ?
If yes, you will get the following error: ` wrong number of arguments (given 2, expected 3)`